lib/components/util/prefersReducedMotion.tsx

```diff
@@ -0,0 +1,9 @@
+/**
+ * Identifies whether the user's machine has "reduced motion" enabled
+ *  in their local settings. If reduced motion is on, the app should
+ * show as few animations & transitions as possible.
+ * @returns boolean reflecting whether user prefers reduced motion
+ */
+export const prefersReducedMotion = window.matchMedia(
+  '(prefers-reduced-motion: reduce)'
```

Comment:
It is also possible to check for this in the CSS directly https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Reply:
Yes but since we're using `setTimeout`, I think we need it here too!